recipe/patches/0008-bpo-22699-Allow-compiling-on-debian-ubuntu-with-a-di.patch

```diff
@@ -45,7 +46,7 @@ index db4b8ed031..d3661bccdf 100644
 +        tmpfile = os.path.join(self.build_temp, 'multiarch')
 +
          ret = run_command(
-             '%s -print-multiarch > %s 2> /dev/null' % (cc, tmpfile))
+             '%s -print-multiarch > %s 2> /dev/null' % (CC, tmpfile))
```

Comment:
This should probably be lowercase
```suggestion
             '%s -print-multiarch > %s 2> /dev/null' % (cc, tmpfile))
```

Reply:
```
+        cc = sysconfig.get_config_var('CC')
```

Reply:
I was prompted to fix patch 0008, and I elected to incorporate both from head and commit

Reply:
That's why I noted above to pay attention to patch no. 0008 in case there are issues that aren't clear to me. I think it should be okay, but I someone should review it

Reply:
Applied it as I had to lowercase it also the last time when did update this patch.

Reply:
Yes, but last time this line `cc = sysconfig.get_config_var('CC')` wasn't there. It's okay, as long as it passes :) 

Thank you for taking care of this!!!